File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitTimes2PlusJ.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

import static net.fwitz.math.binary.split.SplitComplex.splitComplex;

public class SplitTimes2PlusJ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("f(z) = z * (2 + j)")
                .computeFn(z -> z.times(splitComplex(2, 1)))
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitTimes2PlusJ.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitTimes2PlusJ {
    val TWO_PLUS_J = SplitComplex(2, 1)

    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("f(z) = z * (2 + j)")
        .computeFn { z -> z * TWO_PLUS_J }
        .render()
}